recipe/meta.yaml

```diff
@@ -40,18 +38,29 @@ test:
 about:
   home: http://had.co.nz/reshape
   license: MIT
-  summary: 'Flexibly restructure and aggregate data using  just two functions: melt and cast.'
+  summary: 'Flexibly restructure and aggregate data using just two functions: melt and cast.'
   license_family: MIT
-
   license_file:
-    - {{ environ["PREFIX"] }}/lib/R/share/licenses/MIT
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
```

Comment:
The quotes will be removed by the updater bot again, may as well leave them off.

Reply:
I am using the helper script with conda-build 3.21.7 to perform bulk updates. If we don't want the license file to be enclosed in strings, we need to either make the change to the cran skeleton in conda-build or the helper script.